### ISSUE TITLE
wishlist: use synchronized settings instead of local settings, maybe optionally.

### ISSUE THREAD

**Issue by slippycheeze** [luben/load-progress-bar#14]
Right now this addon uses `browser.storage.local` for settings.  Using instead the `browser.storage.sync` system the configuration on one computer will automatically sync with other computers if Firefox sync is enabled.

That'd be rather nice to have, perhaps as an option, since this is a "chrome-like" setting, and Firefox syncs most other UI / chrome settings automatically too.

**Comment by luben** [comment]
Good idea, I didn't know about the sync storage. Will experiment and release something in a few days.

**Comment by luben** [comment]
I researched and experimented a little bit this weekend. Having a local/sync storage fallback adds noticeable latency as the extension have to lookup its config for each page it is loaded into. I also looked about dropping support for the `storage.local` but I think it's not the right moment - the `storage.sync` was introduced with FF v63, the old ESR is v60, there is new ESR v68 just released but there are still some users on v60 and a few on older versions, e.g. ESR v52.

So I will leave it open for now and revisit later.
